lib/src/simple_op_store.rs

```diff
@@ -212,12 +212,12 @@ impl OpStore for SimpleOpStore {
             let hex_prefix = prefix.hex();
             if hex_prefix.len() == OPERATION_ID_LENGTH * 2 {
                 // Fast path for full-length ID
-                if matches_root || op_dir.join(hex_prefix).try_exists()? {
+                return Ok(if matches_root || op_dir.join(hex_prefix).try_exists()? {
```

Comment:
I'm ~Ok with this here, but I like the original a bit better, and I don't want this to be enforced by clippy.